app/services/cocina_version_validator.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+# Checks that the major and minor cocina versions match.
+class CocinaVersionValidator
```

Comment:
I don't think we want this.  It was important in SDR-API where we had less control over the clients, but here we only have one party (DSA) sending requests.